core-services/sync-rover-groups/_config.yaml

```diff
@@ -13,7 +13,7 @@ cluster_groups:
 groups:
   aos-qe:
     clusters:
-    - app.ci
+    - apps.ci
```

Comment:
@smg247 Since this private URL is https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/, I think here should be `apps.ci`, right?

Reply:
ok, thanks!

Reply:
@smg247 One more question, now the job [openshift-openshift-tests-private-master-presubmits.yaml](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-presubmits.yaml#L8) uses the `build02` cluster, do I need to add `build01 - build04` here? Or this build farm cluster will be changed to `app.ci` automatically once this PR is merged? Thanks!

Reply:
If no need, could you help add `/lgtm` label? Thanks!

Reply:
The job itself will run on the build02 cluster, but you will access the logs using the new qe-private-deck instance running on app.ci. There is no need to sync the group to any of the other clusters.